### DISCUSSION TITLE
Coordinate system

### DISCUSSION THREAD

**Discussion by bhnizar** [Dlubal-Software/RFEM_Python_Client#367] [Q&A] [answered]
Hello RFEM community,
I would like to know if it is possible to modify the direction of the global z-axis upwards.
In the same way, is it possible to modify the direction of the local axes upwards for structural elements such as members?
Could you please help me?

**Comment by heetrojivadiya** [comment, marked as answer]
Hey @bhnizar,

Thank you for reaching out to us. Yes, it is possible with `BaseSettings()`. 

```py
from RFEM.initModel import Model
from RFEM.enums import GlobalAxesOrientationType, LocalAxesOrientationType
from RFEM.baseSettings import BaseSettings

Model(True, 'Demo1')
BaseSettings(global_axes_orientation=GlobalAxesOrientationType.E_GLOBAL_AXES_ORIENTATION_ZUP, local_axes_orientation=LocalAxesOrientationType.E_LOCAL_AXES_ORIENTATION_ZUP)
```

![image](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/108512789/2b86b414-57dc-46c4-9340-7b2dd143d18a)

I hope this will work for you.

**Reply by bhnizar** [reply]
Thank you! It's great to know that the solution worked effectively.
